src/overlays/actors/ovl_En_Ig/z_en_ig.c

```diff
@@ -531,13 +562,13 @@ s32 func_80BF1DF4(EnIg* this, GlobalContext* globalCtx, struct_80133038_arg2* ar
             Math_Vec3f_Copy(&this->actor.world.pos, &sp40);
 
             if (ABS_ALT(BINANG_SUB(this->actor.world.rot.y, sp50->actor.shape.rot.y)) <= 0x4000) {
-                this->unk_2A4 = -75;
+                *((s8*)&this->unk_298.hasResult) = -75; // ???
```

Comment:
This field is accessed very oddly. It needs to be treated as an `s8` to match, but it needs to be a `s32` to match in `z_schedule`. It also isn't used in a way that makes sense here. I'm not sure what's going on.

Reply:
Looking at it, I don't think unk_298, is actually a `ScheduleResult`. Instead it looks to be just separate fields of the form 
```c
/* 0x298 */ u8 scheduleResult,
/* 0x29C */ s32* unk_29C // Message script?
/* 0x2A0 */ s32 unk_2A0 // Message related
/* 0x2A1 */ s8 unk_2A4 // EnDoor related
```

Reply:
Ah yeah, taking another look at it, that's completely write. I thought the struct in the actor struct was what was being passed into `Schedule_RunScript`, but it's just a stack variable.